app/lib/actions/katello/content_view_version/incremental_update.rb

```diff
@@ -213,44 +222,23 @@ def run # rubocop:disable Metrics/CyclomaticComplexity
           base_repos = ::Katello::ContentViewVersion.find(input[:old_version]).repositories
           new_repos = ::Katello::ContentViewVersion.find(input[:new_content_view_version_id]).repositories
 
-          if input[:copy_action_outputs].present? && input[:copy_action_outputs].last[:pulp_tasks].present?
-            if input[:copy_action_outputs].last[:pulp_tasks].last[:pulp_href]&.include?("/pulp/api/v3/")
-              new_repos.each do |new_repo|
-                matched_old_repo = base_repos.where(root_id: new_repo.root_id).first
```

Comment:
I hit an issue:

```
 Error:

NoMethodError

undefined method `errata' for nil:NilClass

---
- "/home/vagrant/katello/app/lib/actions/katello/content_view_version/incremental_update.rb:229:in
  `block in run'"
- "/home/vagrant/foreman/.vendor/ruby/2.7.0/gems/activerecord-6.0.4.7/lib/active_record/relation/delegation.rb:88:in
  `each'"
- "/home/vagrant/foreman/.vendor/ruby/2.7.0/gems/activerecord-6.0.4.7/lib/active_record/relation/delegation.rb:88:in
  `each'"
- "/home/vagrant/katello/app/lib/actions/katello/content_view_version/incremental_update.rb:226:in
```

From a quick debug I found my `matched_old_repo` was `nil`.  In my case, I was incrementally updating an older content view version that didn't include a repository that I added later.  So, my `new_repo` was defined, but not the old one.

I think these testing steps should reproduce the issue:
1) Have a component CV with one filtered repo
2) Publish the component
3) Publish the composite
4) Add a new repo to the component CV and publish again
5) Publish the composite CV again for good measure
6) Now, promote the old 1-repo composite CV version to Library
7) Incrementally update the old 1-repo component CV version with `--propagate-all-composites` and see the error.